src/entities/transactions/eip1559TransactionType.ts

```diff
@@ -0,0 +1,4 @@
+export enum EIP1559TransactionType {
```

Comment:
if this is imported into a non-TS file (such as SpeedUpAndCancel) it should complain - you may need to do something similar to entites/assetTypes where you export both an enum as well as the plain old JS version